R/get_inboveg_layer_information.R

```diff
@@ -0,0 +1,123 @@
+#' @title Query layer information of recordings (relevé) from INBOVEG
+#'
+#' @description This function queries the INBOVEG database for layer
+#' information on recordings  for one or more surveys.
+#'
+#' @param survey_name A character string or a character vector, depending on
+#' multiple parameter, giving the name or names of the survey(s) for which you
+#' want to extract recordings information. If missing, all surveys are returned.
+#' @param connection dbconnection with the database 'Cydonia'
+#' on the inbo-sql07-prd server
+#' @param multiple If TRUE, survey_name can take a character vector with
+#' multiple survey names that must match exactly. If FALSE (the default),
+#' survey_name must be a single character string (one survey name) that can
+#' include wildcards to allow partial matches
+#'
+#' @return A dataframe with variables Name (of the survey), RecordingGivid (unique Id),
+#' UserReference, LayerCode, LayerDescription, CoverCode, Coverpercentage and Mean hight (cm)
+#'
+#' @importFrom glue glue_sql
+#' @importFrom DBI dbGetQuery
+#' @importFrom assertthat assert_that
+#'
+#' @export
+#' @family inboveg
+#' @examples
+#' \dontrun{
+#' library(inbodb)
+#' library(DBI)
+#' library(odbc)
```

Comment:
As you are only using functions of inbodb, these packages don't have to be loaded.  I also suppose dplyr doesn't have to be loaded in this case?  (Could you please try? Restart R, only load inbodb and try the example.)  Certainly in an example, I think it is important to stick to the necessary steps and not confuse users by doing unnecessary steps.
```suggestion
```